viz/charts/linkerd-viz/values.yaml

```diff
@@ -204,6 +204,8 @@ tap:
       # -- Amount of ephemeral storage that the tap container requests
       request: ""
 
+  ignoredHeaders:
```

Comment:
We usually use infinitives for the settings, so I'd prefer `ignoreHeaders`. Also please provide a default empty array `[]` and a comment documenting the setting so it can be picked up by the chart README file.